front/src/iframe_api.ts

```diff
@@ -166,6 +168,15 @@ window.WA = {
         }, '*');
     },
 
+    exitSceneTo(url : string) : void{
```

Comment:
Could you rename this to 'goToRoom'?

Reply:
done

Reply:
the idea was to hint developers that it does the same as exitSceneUrl